{"uuid": "511c40a5-7ff9-4202-957f-f47f21f3d7a8", "children": ["2c986460-1559-49ce-a0bd-f6629998ab35"], "befores": [{"name": "create_section", "status": "broken", "statusDetails": {"message": "KeyError: 'id'\n", "trace": "  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 910, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/berserker/python/api_automation_2024/todo_api/conftest.py\", line 48, in create_section\n    id_section_created = section_created[\"body\"][\"id\"]\n                         ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^\n"}, "start": 1712608493152, "stop": 1712608493757}], "start": 1712608493152, "stop": 1712608493779}